libs/filameshio/src/MeshReader.cpp

```diff
@@ -159,30 +159,32 @@ static size_t fileSize(int fd) {
 
 namespace filamesh {
 
-MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine* engine, const utils::Path& path,
-        MaterialRegistry& materials) {
-
+MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine *const engine, const utils::Path& path,
+        MaterialRegistry& materialRegistry) {
     Mesh mesh;
 
-    int fd = open(path.c_str(), O_RDONLY);
+    /* Open the mesh file for reading only. */
+    const int fd = open(path.c_str(), O_RDONLY);
+
+    /* Check if an error occurred when opening the file. */
+    if (fd == -1) return mesh;
 
-    size_t size = fileSize(fd);
-    char* data = (char*) malloc(size);
-    read(fd, data, size);
+    const size_t size = fileSize(fd);
+    const char *const data = (char *)malloc(size);
```

Comment:
@pixelflinger We should decide what style we want for pointers declarations. It seems most of our casts are `(char*) foo` (not space before *, one space after the cast), but not always.

Reply:
it seems to me the cast should follow the same style as a variable declaration. Since we use:

`char *value;` and not `char* value;` in most of our code, I think a cast should be `(char *)` as well.  It seems to me we usually don't put a space after the cast (this file did, but that seems to be an outlier, I think).

so, I think `(char *)value` is probably the style I would go with.